### DISCUSSION TITLE
Merc AutoEquip Tier - Found, Equipped, Tossed Andariels

### DISCUSSION THREAD

**Discussion by jh86** [isid0re/SoloLeveling#387] [Q&A]
![andarielsfiasco](https://user-images.githubusercontent.com/51974054/153551024-01b48549-901a-4b5a-b060-08cbb9ec8cac.png)

Andariels was listed as value of 50,000 in the MercTier listing and it ended up with one worth 89.7. Did the negative res freak out the bot? Item definitely got tossed as a result. I'll find another, would like to know why it chucked it though.

**Comment by isid0re** [comment]
I think I know why. I had an issue earlier with shako being replaced when it was socketed and um'ed

The cause was a lack of inventory space when it removed the item to socket. So it dropped it. I thought I had fixed that for both merc and bot. But since you've mentioned it I will double check the various function files
Itemoverrides.js
Quest.js
Townoverrides.js

**Reply by jh86** [reply]
Thanks for looking into that. There was one more thing that happened last night:
![enigmafiasco](https://user-images.githubusercontent.com/51974054/153595454-9c129530-1e14-4a68-bfaa-528bfaf83a35.png)
![enigma2](https://user-images.githubusercontent.com/51974054/153595837-a7fad763-b2c4-4e25-b411-69d1bdc69a19.png)

Last night the druid found an acceptable base for enigma, made it, then 20 minutes later "rerolled" it which actually means just clearing it and keeping the base. It also did a respec at that time but only gave itself 30 dex which is not enough to continue using the HoTo that it made in a knout (I after the fact edited the scripts to only use Flails, with the exception of the druid who had made the knout, his I left alone) so he is now using a vipermagi and no main hand weapon lol)

**Reply by jh86** [reply]
I had a look in the druid.windbuild.js and under stats there's no allocation for dexterity. I have added it and set to 35 to be able to use a HoTO but It has already used all of its respecs. Is there a way to make the bot start creating Tokens from the bosses so I can rectify this? I had it using a ravenfrost at some point which allowed the use of the HoTO but now it's sitting in the stash. The finalbuild items also have no dexterity so that wouldn't solve this problem.

**Reply by isid0re** [reply]
Sure to enable tokens you need to add the following cubing recipe to the druid.sololeveling.js 

`Config.Recipes.push([Recipe.Token]);` 

Edit: also you will want to add the pickit line to keep the token of absolution if not already included in your pickits

**Reply by isid0re** [reply]
Also looks like I need to go thru all the finalbuilds and update stats. For builds. Thanks

**Comment by isid0re** [comment]
Just updated druid builds for dexterity. 
Thanks for posting
